Adyen/Helpers/UIViewConstraintsHelper.swift

```diff
@@ -54,5 +97,32 @@ extension AdyenScope where Base: UIView {
         anchor(inside: containerView, with: insets)
         return containerView
     }
+}
 
+/// Edges combined with their inset values.
+/// :nodoc:
+public enum AnchorEdge: Hashable {
```

Comment:
should these be `.leading` and `.trailing` instead of `.left` and `.right`?

Reply:
I wasn't sure either. While on similar topic, should we namespace these in somewhere?

Reply:
maybe let's look from a different perspective. do we need this `enum` at all, or can we use `EdgeInsets` for this?

Reply:
This is the result of my intention to improve the one we have with UIEdgeInsets. Instead of providing edges and paddings differently.
One other approach was to provide CGFloat as optional, but then again, it won't be `UIEdgeInsets`